README.md

```diff
@@ -89,6 +89,20 @@ npm start
 ]
 ```
 
+* `CONNECTOR_ROUTES` (default: `[]`) Explicitly add routes to the connector. If `prefix` is the most specific
+  route name that matches a destination, then the payment will be forwarded to `connector`. ex:
+```js
+[
+  {
+    "prefix": "", // matches any destination
+    "connector": "ilpdemo.red.connie"
```

Comment:
And, for clarity, rename the `prefix` field (e.g. `destinationPrefix` or `targetPrefix`).